### ISSUE TITLE
Guide for viewing the same channel for multiple files

### ISSUE THREAD

**Issue by rafmudaf** [ebranlard/pyDatView#63]
Is there such a guide? If not, can you summarize here and I'll work to add documentation?

**Comment by rafmudaf** [comment]
I figured it out and added an additional line of documentation to the README
https://github.com/rafmudaf/pydatview

**Comment by ebranlard** [comment]
Thanks, feel free to do a pull request. I should definitely improve on the documentation..
